src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -126,9 +124,51 @@ const FilterLabel = styled.label`
   font-size: 15px;
   font-style: normal;
   font-weight: 500;
-  line-height: 17px; /* 113.333% */
+  line-height: 17px;
   letter-spacing: 0.15px;
 `;
+const Leftheader = styled.div`
```

Comment:
The styles seems to be getting large it might make sense to make an `OrgHeaderStyles.tsx` file in the `./src/pages/tickets/org/orgHeader` folder to make this file a bit more compact